src/Network/Algorand/Node/Api.hs

```diff
@@ -249,3 +247,18 @@ instance Mime.MimeRender Binary BSL.ByteString where
 
 instance Mime.MimeRender Binary [SignedTransaction] where
   mimeRender _ = mconcat . map (MP.pack . MP.Canonical)
+
+-- | Content type for the `v2/blocks` endpoint.
```

Comment:
There are many other endpoints that can return data in msgpack format. AFAIU we just don't support them yet. Can we update `Mime.MimeUnrender MsgPack` instance to be defined not just for `BlockWrapped` but for something more general (anything that can be decoded from msgpack) and update this comment saying that it's for any endpoint that may return msgpack?

Reply:
I think we can also move content types to its own module

Reply:
Yes, good idea.

Reply:
> Do you know where to find `block` structure in the official docs?

And the source of truth is here:
https://github.com/algorand/go-algorand/blob/master/daemon/algod/api/server/v2/handlers.go
https://github.com/algorand/go-algorand/blob/master/daemon/algod/api/server/v2/generated/types.go